scan-triggers/aws-python-bucket-full-and-scheduled-scan/README.md

```diff
@@ -44,16 +44,16 @@ It deploys all the required resources to trigger a full scan, scheduled or not,
 
 ## Running it
 
-After deployment, trigger the new Lambda function `BucketFullScan` with any payload, empty or otherwise, using whatever method you feel most comfortable with. The scan will be queued right after that.
+After deployment, trigger the new Step Function `FullScanStarterLoopStepFunction` with any payload, empty or otherwise, using whatever method you feel most comfortable with. The scan will start right after that.
```

Comment:
```suggestion
After deployment, trigger the new Step Function `FullScanStarterLoopStepFunction` with any payload, empty or otherwise, using whatever method you feel most comfortable with. The scan starts immediately..
```